sdk/spring/azure-spring-boot-bom/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-spring-boot-bom</artifactId>
+    <version>2.2.3-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>Azure Spring Boot BOM</name>
+    <description>BOM for Microsoft Azure Spring Boot Support</description>
+    <url>https://github.com/Microsoft/azure-spring-boot</url>
+
+    <licenses>
+        <license>
+            <name>MIT</name>
+            <url>https://github.com/Microsoft/azure-spring-boot/blob/master/LICENSE</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>ZhijunZhao</id>
+            <name>Zhijun Zhao</name>
+            <email>zhijzhao@microsoft.com</email>
+        </developer>
+
+        <developer>
+            <id>yungez</id>
+            <name>Yunge Zhu</name>
+            <email>yungez@microsoft.com</email>
+        </developer>
+    </developers>
+
+    <organization>
+        <name>Microsoft</name>
+        <url>https://www.microsoft.com</url>
+    </organization>
+
+    <scm>
+        <connection>scm:git:git://github.com/Microsoft/azure-spring-boot-starters.git</connection>
+        <developerConnection>scm:git:ssh://github.com:Microsoft/azure-spring-boot-starters.git</developerConnection>
+        <url>https://github.com/Microsoft/azure-spring-boot/tree/master</url>
+    </scm>
+
+    <properties>
+        <azure.spring.boot.version>2.2.3-SNAPSHOT</azure.spring.boot.version>
+        <azure.dependencies.bom.version>2.2.0.M4</azure.dependencies.bom.version>
+        <spring.data.cosmosdb.version>2.2.2</spring.data.cosmosdb.version>
```

Comment:
Same here. v2.2.3 is the latest version out there.

Reply:
Updated.